lib/saito/network.js

```diff
@@ -437,6 +774,14 @@ class Network {
         }
     }
 
+    sendPeerRequestWithCallback(message, data = "", peer) {
+        for (let x = this.peers.length - 1; x >= 0; x--) {
+            if (this.peers[x] === peer) {
+                this.peers[x].sendRequestWithCallback(message, data, callback);
+            }
+        }
+    }
+
     sendRequest(message, data = "") {
```

Comment:
for 4 functions here `sendPeerRequest`, `sendPeerRequestWithCallback`, `sendRequest` & `sendRequestWithCallback`, I think we can simplify just 2 functions as below:
```
sendRequest(message, data = "",callback) {
  for (let x = this.peers.length - 1; x >= 0; x--) {
      callback ? this.peers[x].sendRequest(message, data) : this.peers[x].sendRequest(message, data, callback);
  }
}

sendPeerRequest(message, data = "", peer, isCallback) {
  for (let x = this.peers.length - 1; x >= 0; x--) {
    if (this.peers[x] === peer) {
        isCallback ? this.peers[x].sendRequest(message, data) : this.peers[x].sendRequest(message, data,callback);
    }
  }
}
```